microsoft-edge/devtools-guide-chromium/evaluate-performance/index.md

```diff
@@ -145,39 +143,38 @@ After you measured and verified that the animation isn't performing well, the ne
 
    ![The Main section.](../media/evaluate-performance-performance-main.msft.png)
 
-1. There's a lot of data in the recording.  To Zoom into a single event, click, hold, and drag your cursor over the **Overview**, which is the section that includes the **FPS**, **CPU**, and **NET** charts.  The **Main** section and **Summary** panel only display information for the selected portion of the recording.
+1. There are a lot of data in the recording.  To zoom into a portion of the recording, click, hold, and drag your cursor over the **Overview**, which is the section that includes the **CPU** and **NET** charts.  The **Main** section and **Summary** panel only display information for the selected portion of the recording.
```

Comment:
```suggestion
1. There is a lot of data in the recording.  To zoom into a portion of the recording, drag the **Overview**, which is the section that includes the **CPU** and **NET** charts.  The **Main** section and **Summary** panel only display information for the selected portion of the recording.
```
https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/d/drag-and-drop - "Don't use click and drag."
https://learn.microsoft.com/en-us/style-guide/procedures-instructions/describing-interactions-with-ui - 
See **Move, drag** row